docs/glsl-fuzz-generate.md

```diff
@@ -115,7 +115,7 @@ Assuming you have extracted the `graphicsfuzz.zip` file to get `graphicsfuzz/`:
 
 ```sh
 # Copy the sample shaders into the current directory:
-cp -r graphicsfuzz/shaders/samples samples
+cp -r graphicsfuzz/shaders/src/main/glsl/samples samples
```

Comment:
This is the path from the release zip, so it should be correct, but perhaps we should clarify